roles/generate-jenkins/templates/UNRAID_TEMPLATE.j2

```diff
@@ -50,6 +50,9 @@
     <Privileged>{{ "true" if privileged is sameas true else "false" }}</Privileged>
     <Support>{{ project_github_repo_url }}/issues/new/choose</Support>
     <Shell>bash</Shell>
+{% if project_deprecation_status %}
```

Comment:
I've noticed a few other places where you're doing something like
```
{% if project_deprecation_status = true %}
```
(Not sure if that's the correct syntax)
Is that needed here?

Reply:
I copied your logic here. https://github.com/linuxserver/docker-jenkins-builder/blob/master/roles/generate-jenkins/templates/DOCUMENTATION.j2#L7